entrypoint.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
+set -e
 
-/opt/fahclient/FAHClient \
-    --user=${USER} \
-    --team=${TEAM} \
-    --passkey=${PASSKEY} \
-    --gpu=${ENABLE_GPU} \
-    --smp=${ENABLE_SMP} \
-    --power=full \
-    --gui-enabled=false \
-    ${@}
+# if a command was specified, we run that command. E.g. helpful when we do
+# docker run -it <container> /bin/bash
+if [ "${1}" ]
```

Comment:
Isn't it gonna be triggered by `docker run <image> --some-random-arg=value` as well?

Reply:
:+1: I hadn't considered that.